test/common/async.cpp

```diff
@@ -17,20 +17,23 @@
 #include <ight/common/pointer.hpp>
 #include <ight/common/settings.hpp>
 
+#include <ight/ooni/dns_injection.hpp>
 #include <ight/ooni/http_invalid_request_line.hpp>
+#include <ight/ooni/tcp_connect.hpp>
 
-#include <event2/thread.h>  // XXX
-#include <unistd.h>  // XXX
+#include <unistd.h>
 
 using namespace ight::common::async;
 using namespace ight::common::pointer;
 using namespace ight::common;
 
+using namespace ight::ooni::dns_injection;
 using namespace ight::ooni::http_invalid_request_line;
+using namespace ight::ooni::tcp_connect;
 
 TEST_CASE("The async engine works as expected") {
-    evthread_use_pthreads();
-    ight_set_verbose(1);
+
+    //ight_set_verbose(1);
```

Comment:
Yes, it should be removed. The reason to have this line of code is silly: it makes the default logger verbose, which is helpful when you want to see the noise made by the code in the backfround thread, which -as explained above- uses the default logger.